monorepo/monorepo-ws/apps/articleapp/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:16.14-alpine
+WORKDIR /app
+COPY package.json package-lock.json ./
+ENV PORT=9000
+EXPOSE ${PORT}
+RUN npm install & npm install ajv && npm cache clean --force
```

Comment:
to ajv by tu podle mě nemělo bejt takhle adhoc. 
1) zkusil bych to ještě jednou bez něj, jestli nám to nekravilo kvůli něčemu jinýmu. 
2) pokud je nutný, tak ho dát do dependencí ať je správně zaverzovaný.